frontend/public/tenants/tenant.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Default static file that will be overridden through a ConfigMap for each environment.